packages/babel-helpers/src/helpers/applyDecs2311.ts

```diff
@@ -1,6 +1,8 @@
 /* @minVersion 7.24.0 */
 /* @mangleFns */
 
+/* eslint-disable @typescript-eslint/no-unnecessary-type-assertion -- `typescript-eslint` complains when using `!` */
```

Comment:
It seems to be using the top-level tsconfig.json, and I can't reproduce this issue in the typescript-eslint repl.

I don't know if this is a bug or an issue with our configuration.

Reply:
!s are annoying but ok